src/net/sourceforge/kolmafia/objectpool/Concoction.java

```diff
@@ -471,6 +471,10 @@ public int compareTo(final Concoction o) {
       return 1;
     }
 
+    // Sort helpers by name if more than one helper.
+    if ((this.isHelper() && (o.isHelper()))) {
```

Comment:
you could remove some brackets on this line I think?
```suggestion
    if (this.isHelper() && o.isHelper()) {
```

Reply:
Done